internal/server/auth/model.go

```diff
@@ -0,0 +1,18 @@
+package auth
+
+import (
+	"github.com/golang-jwt/jwt/v4"
+	"time"
+)
+
+type Claims struct {
+	jwt.RegisteredClaims //nolint: typecheck
+	UserID               int
+}
+
+const TokenExp = time.Hour * 3
+
+// SecretKey пока что в коде, т.к. по заданию не понятно где должен храниться он.
+// Если в записать в переменную окружения, то автотесты на сервере не пройдут.
```

Comment:
SUGGESTION: Of course because you need to modify the GitHub workflow.

Reply:
done